app/lib/server/controllers/run.controller.js

```diff
@@ -13,8 +13,15 @@
 
 const { databaseManager: { repositories: { RunRepository } } } = require('../../database');
 
-const listRuns = async (req, res) => res.json(await RunRepository.findAll()); // TODO DTO
+/**
+ * List All runs in db
+ */
+
+const listRunsHandler = async (req, res, next) => {
+    const result = await RunRepository.findAll();
+    res.json(result);
+}; // TODO DTO
```

Comment:
Please add the DTO in this PR to have the full pipeline.
Also, please add two typedefs: one for the endity and one for the model (you can have a look in bookkeeping in `lib/domain/entities` and `lib/database/models/typedefs`). Maybe at some point we'll use typescript... but until then we have to provide ways for the IDE to help us

Reply:
applied